en/docs/integration-guides/integration-service-as-a-managed-api/how-to-expose-an-integration-service-as-a-managed-api.md

```diff
@@ -0,0 +1,160 @@
+            "name":"seth mears",
+            "hospital":"pine valley community hospital",
+            "category":"surgery",
+            "availability":"3.00 p.m - 5.00 p.m",
+            "fee":8000.0
+        }
+    ]
+    ```
+
+### Step 3: Creating Configuration for APIM
+
+1. Click on the `</>` Button.
+
+    <a href="{{base_path}}/assets/img/integration-guides/integration-service-as-a-managed-api/code-button.png"><img src="{{base_path}}/assets/img/integration-guides/integration-service-as-a-managed-api/code-button.png" alt="code button" width="80%"></a>
+
+2. Add `import ballerinax/wso2.apim.catalog as _;` after the existing imports, then click the File View icon in the top-left corner.
+
+    <a href="{{base_path}}/assets/img/integration-guides/integration-service-as-a-managed-api/import.png"><img src="{{base_path}}/assets/img/integration-guides/integration-service-as-a-managed-api/import.png" alt="import button" width="80%"></a>
+
+3. Select `Ballerina.toml` file and add `remoteManagement=true` after the existing `[build-options]`.
+
+    <a href="{{base_path}}/assets/img/integration-guides/integration-service-as-a-managed-api/build-options.png"><img src="{{base_path}}/assets/img/integration-guides/integration-service-as-a-managed-api/build-options.png" alt="build options" width="80%"></a>
+
+4. Use the BI Configurations `+` icon to add the following configurations, and save each of them.
+
+    <a href="{{base_path}}/assets/img/integration-guides/integration-service-as-a-managed-api/ui-config.png"><img src="{{base_path}}/assets/img/integration-guides/integration-service-as-a-managed-api/ui-config.png" alt="configurations" width="80%"></a>
+
+    ```conf
+    [ballerinax.wso2.apim.catalog]
+    serviceUrl="<Url of the service catalog endpoint>"
+    tokenUrl="<Url of the token endpoint>"
+    username="<username>"
+    password="<password>"
+    clientId="<clientId>"
+    clientSecret="<Client secret>"
+    ```
+
+    !!! Info
+        See the [Create a Rest API](https://apim.docs.wso2.com/en/3.2.0/learn/design-api/create-api/create-a-rest-api/) documentation to learn how to get configurations.
+
+        These are some default values, but `clientId` and `clientSecret` should be replaced with your own.
+
+        ```conf
+        [ballerinax.wso2.apim.catalog]
+        serviceUrl="https://127.0.0.1:9443/api/am/service-catalog/v1"
+        tokenUrl="https://localhost:9443/oauth2/token"
+        username="admin"
+        password="admin"
+        clientId="<clientId>"
+        clientSecret="<Client secret>"
+        ```
+
+5. Let's start the API Manager runtime before starting the WSO2 Integrator: BI.
+
+    1.  Download and set up [WSO2 API Manager](https://wso2.com/api-management/).
+    2.  Start the [API-M server](https://apim.docs.wso2.com/en/latest/install-and-setup/install/installing-the-product/running-the-api-m/).
+
+### Step 4: Run the application
+
+1. Select the BI Extension icon and click the `Run` button to execute.
```

Comment:
```suggestion
1. Select the BI Extension icon and click the **Run** button to execute.
```

Reply:
All the UI elements and button texts etc should be in bold color. Please check and fix all the places.